### PR TITLE
chore: add shared workflow for processing nix flake files

### PR DESCRIPTION
**Pull request by dpal**
This pull request is generated the workflow maintenance script.

### CODE REVIEW

**Review by rvolosatovs — CHANGES_REQUESTED** [review]
I think this belongs in `.github/workflows/nix-update.yml`

File: .github/workflows/shared-nix-update.yml [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+        with:
+          branch: nix-update
+          commit-message: "build(nix): update flake lock"
+          title: "build(nix): update flake lock"
+          body: ${{ steps.nix-update.outputs.pull-request-body }}
+          labels: dependencies, nix
+          assignees: |
+            rvolosatovs
+          signoff: true
+
```
Comment (rvolosatovs) [inline .github/workflows/shared-nix-update.yml:27]:
```suggestion
```

File: .github/workflows/shared-nix-update.yml [hunk 2]
```diff
@@ -0,0 +1,27 @@
+name: Nix update
```
Comment (rvolosatovs) [inline .github/workflows/shared-nix-update.yml:1]:
```suggestion
name: Update Nix flake
```

File: .github/workflows/shared-nix-update.yml (hunk 1 above)
Comment (dpal) [reply .github/workflows/shared-nix-update.yml]: Addressed

File: .github/workflows/shared-nix-update.yml (hunk 2 above)
Comment (dpal) [reply .github/workflows/shared-nix-update.yml]: Addressed

**Review by rvolosatovs — CHANGES_REQUESTED** [review]
I still think this should be in `.github/workflows/nix-update.yml`, just like other shared workflows in this repo

**Review by rvolosatovs — APPROVED** [review]

**Pull request merged**